cmd/machine-config-daemon/pivot.go

```diff
@@ -381,6 +253,7 @@ func run(_ *cobra.Command, args []string) error {
 func Execute(cmd *cobra.Command, args []string) {
 	err := run(cmd, args)
 	if err != nil {
-		glog.Fatalf("%v", err)
+		fmt.Printf("error: %v\n", err)
+		os.Exit(1)
```

Comment:
Any reason for this (other than that we should move from glog to klog 🙂 )?

Reply:
`Fatalf` dumped a stack trace which wasn't necessary here.

Reply:
This was noted in the commit message from 0bc206007612159988721f39f47dd8f12d3ffb25
>  (Also change the toplevel to not spew a stack trace on error)